code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -1,3 +1,5 @@
+#define SPEAK(message) radio.talk_into(src, message, radio_channel, get_spans())

```

Comment:
The radio.talk_into system is extremely picky about HOW and IF it wants to work.